zerver/openapi/zulip.yaml

```diff
@@ -21909,6 +21926,17 @@ components:
             Is `null` for channels with no message history.
 
             **Changes**: New in Zulip 2.1.0.
+        is_recently_active:
+          type: boolean
+          description: |
+            Whether the channel has recent message activity. Clients should use this to implement
+            [sorting inactive channels to the bottom](/help/manage-inactive-channels)
+            if `demote_inactive_streams` is enabled.
+
+            Previously, clients implemented the demote_inactive_streams from local message history,
+            resulting in a choppy loading experience.
+
+            **Changes**: New in Zulip 10.0 (feature level 323)
```

Comment:
"Previously" text always goes inside the "Changes" paragraph. Also, it should appear in all endpoints referencing a parameter.

``` diff
diff --git a/zerver/openapi/zulip.yaml b/zerver/openapi/zulip.yaml
index 200f2174c8..6d39b5b726 100644
--- a/zerver/openapi/zulip.yaml
+++ b/zerver/openapi/zulip.yaml
@@ -21933,10 +21933,9 @@ components:
             [sorting inactive channels to the bottom](/help/manage-inactive-channels)
             if `demote_inactive_streams` is enabled.
 
-            Previously, clients implemented the demote_inactive_streams from local message history,
-            resulting in a choppy loading experience.
-
-            **Changes**: New in Zulip 10.0 (feature level 323)
+            **Changes**: New in Zulip 10.0 (feature level 323). Previously, clients implemented the
+            demote_inactive_streams from local message history, resulting in a choppy loading
+            experience.
         is_announcement_only:
           type: boolean
           deprecated: true
@@ -22946,9 +22945,12 @@ components:
         is_recently_active:
           type: boolean
           description: |
-            Whether the channel has recent message activity. Clients should use this to implement [sorting inactive channels to the bottom](/help/manage-inactive-channels).
+            Whether the channel has recent message activity. Clients should use this to implement
+            [sorting inactive channels to the bottom](/help/manage-inactive-channels).
 
-            **Changes**: New in Zulip 10.0 (feature level 303)
+            **Changes**: New in Zulip 10.0 (feature level 323). Previously, clients implemented the
+            demote_inactive_streams from local message history, resulting in a choppy loading
+            experience.
         stream_weekly_traffic:
           type: integer
           nullable: true
```